@xen-orchestra/lite/src/App/TabConsole.tsx

```diff
@@ -40,11 +46,22 @@ const TabConsole = withState<State, Props, Effects, Computed, ParentState, Paren
         window.dispatchEvent(new UIEvent('resize'))
       },
     },
+    computed: {
+      vm: (state, { vmId }) => state.objectsByType.get('VM')?.get(vmId) as Vm,
```

Comment:
Doesn't TypeScript already know that this is a `Vm`? Thanks to this:
https://github.com/vatesfr/xen-orchestra/blob/cec09f3e7f24283c8ad919136bbfdf735e5baf5f/%40xen-orchestra/lite/src/libs/xapi.ts#L57-L60